### PR TITLE
Bump environments default versions

### PR DESCRIPTION
**Pull request by dincho**

### CODE REVIEW

**Comment by dincho** [comment]
Note for reviewers `check_terraform_changes` is meant to "fail" if there is changes in the infrastructure by the PR. This PR is indeed introducing changes.

File: terraform/environments/main.tf [hunk 1]
```diff
@@ -128,7 +128,7 @@ module "aws_deploy-ap-southeast-1" {
   ami_name      = "aeternity-ubuntu-16.04-v1549009934"
 
   aeternity = {
-    package = "https://releases.ops.aeternity.com/aeternity-2.4.0-ubuntu-x86_64.tar.gz"
+    package = "https://releases.ops.aeternity.com/aeternity-3.0.0-rc.1-ubuntu-x86_64.tar.gz"
```
Comment (lucafavatella) [inline terraform/environments/main.tf]: For all items with `3.0*`, please comment on bootstrap_version: I see v2.0.1 though maybe you meant [v2.2.0](https://github.com/aeternity/infrastructure/compare/v2.0.1...v2.2.0).

Comment (dincho) [reply terraform/environments/main.tf]: good point, I updated all environments bootstrap version to 2.2.0 in 2612809

**Comment by dincho** [comment]
Closing in favour of:
- https://github.com/aeternity/terraform-aws-mainnet/pull/2
- https://github.com/aeternity/terraform-aws-testnet/pull/1
- https://github.com/aeternity/terraform-aws-mainnet-api/pull/1
- https://github.com/aeternity/terraform-aws-devnet/pull/1

**Pull request closed without merging**